lib/wallet/src/Cardano/Wallet/DB/Layer.hs

```diff
@@ -512,7 +521,7 @@ newDBLayerWith _cacheBehavior _tr ti SqliteContext{runQuery} = do
 
     -- FIXME LATER during ADP-1043:
     --   Handle the case where loading the database fails.
-    walletsDB_ <- runQuery $ loadDBVar mkStoreWallets
+    walletsDB__ <- runQuery $ loadDBVar mkStoreWallets
```

Comment:
Wow, I am definitely missing some underscores in my head_ :) 😶‍🌫️

Reply:
I have renamed it to `walletsDB`, which became possible by removing the field import for `DBLayer`.